### ISSUE TITLE
App version 2.1.0 automatically crashed (closed) on joining to call

### ISSUE THREAD

**Issue by kt315ua** [mattermost/mattermost-mobile#7169]
#### Summary
Create call and joint to this call from app.

Result: 
App unexpectedly is closed (crashed?)

### Environment Information
  - Device Name: Samsung Galaxy S10e
  - OS Version: Android 12, One UI 4.1
  - Mattermost App Version: 2.1.0 (Build 6000457)
  - Mattermost Server Version: 7.8.0
  - Database: postgres
  - Database Shema Version: 101

#### Steps to reproduce
Join to the call

#### Expected behavior
- successfully connected to the call
- application not crashed
- enjoy call :)

#### Observed behavior (that appears unintentional)
Application just closed

#### Possible fixes
None

#### NOTE
Sometimes, I can initiate call, and on screen looks good, but when someone joined by desktop app to this call, app is closed - main bug.
Sometimes, when i'm initiating a call from phone i see error: 
```
Error: unable to connect to the voice call: timed out for peer connection 
```

[mattermost_app_samsung_s10e.txt](https://github.com/mattermost/mattermost-mobile/files/10843423/mattermost_app_samsung_s10e.txt)

![photo_2023-02-27_21-37-24](https://user-images.githubusercontent.com/107305429/221665459-8caceca9-84b6-43d6-939e-615887f950f8.jpg)
![photo_2023-02-27_21-42-20](https://user-images.githubusercontent.com/107305429/221666415-211a814d-d1c1-4fa4-be39-667e8eade762.jpg)

**Comment by amyblais** [comment]
@kt315ua Is this same issue as https://github.com/mattermost/mattermost-mobile/issues/7155?

**Comment by kt315ua** [comment]
@amyblais 
Oh, yes. 
sorry for duplicate!
Closed as duplicate #7155
